draft-ietf-rats-corim.md

```diff
@@ -1816,6 +1864,13 @@ compared with the next entry of the Accepted Claims Set array.
 If all entries of the Reference Values array match a corresponding entry in the Accepted Claims Set array, then the `cryptokeys` Reference Value matches.
 Otherwise, `cryptokeys` does not match.
 
+##### Comparison for Integrity Registers {#sec-cmp-integrity-registers}
+
+For each Integrity Register entry in Evidence, the Verifier will use the associated identifier to look up the matching Integrity Register entry in the Reference Value.
```

Comment:
Given the extensible semantics of the word "matching", I would recommend referencing {{sec-cmp-digests}} here instead of 2 sentences from now for the definition of "matching [...] entry".

For each Integrity Register entry `id => digests_{ref}` in the Reference Value, the Verifier will claim a "match" with Evidence if the Evidence's integrity register entry `id => digests_{ev}` compares positively with the method defined in {{sec-cmp-digests}}. If `id` is not present in the Evidence, then the Reference Value does not "match".

Reply:
```suggestion
When asserted as Evidence, the Verifier will accept each Integrity Register entry into the ACS.
When asserted as Reference Values, all of the Integrity Register entries of the matching EMT MUST have a corresponding Integrity Register entry in the ACS Evidence.
Additionally, the Verifier will use the associated Integrity Register label (i.e.,`integrity-register-id-type-choice`) to match the label in a corresponding Integrity Register entry in the ACS.
The Verifier compares the labels which MUST exactly match.
If the labels match, then the Integrity Register digest arrays are compared (see {{sec-comid-integrity-registers}} and {{sec-comparison-for-digests-entries}}).
```
Note: {#sec-comparison-for-digets-entries} needs to be added.

Reply:
@nedmsmith : This text is not clear. We should keep it at a high level describing just the Evidence and Reference Value requirements. Bringing EMT and ACS into mix has made it more complicated, given that we are towards simplification, so this is too much detail without adding much value

Reply:
I am happy with @andrew-draper suggested `text` here. It is also `sort` of aligned with RATS specification: As mentioned in * Section  8.3 of RFC 9334
`
8.3.  Reference Values

 Reference Values used in appraisal procedures come from a Reference
 Value Provider and are then used by the Verifier to compare to
 Evidence.  Reference Values with matching Evidence produce acceptable
 Claims.  Additionally, an appraisal policy may play a role in
 determining the acceptance of Claims.
`

Reply:
> We should keep it at a high level

We already document the high level rules in earlier sections. It doesn't add value to repeat it here. The reader is potentially uncertain how a verifier will process the array of measurements as each measurement could be processed independently. Alternatively, the collection of measurements could be processed as a singleton measurement. The current text provides no insight or guidance that resolves the reader's confusion.

Reply:
I quite liked the crisp nature of text @deeglaze added, so tried to modify her text slightly to specify more clarity on 
matching semantics and attempted purely to see how this expresses. See https://github.com/ietf-rats-wg/draft-ietf-rats-corim/pull/198/files

Reply:
> > We should keep it at a high level
> 
> We already document the high level rules in earlier sections. It doesn't add value to repeat it here. The reader is potentially uncertain how a verifier will process the array of measurements as each measurement could be processed independently. Alternatively, the collection of measurements could be processed as a singleton measurement. The current text provides no insight or guidance that resolves the reader's confusion.

@nedmsmith : I do not think, we have so far introduced any section as to how `Contents` within Integrity Registers (within Evidence and Reference Values) are to be compared. All I was stating was that: We should not describe the loading of Evidence and  RefValue into ACS specific actions here  and just describe the direct matching semantics of the Registers.

Reply:
> We should not describe the loading of Evidence and RefValue into ACS

Line 1869 of my proposed edit could be removed based on this objective. Lines 1870-1873 are aimed at describing the direct matching semantics. Possibly there is better wording, but it seems to say what we want it to say.